playbook/app/pb_kits/playbook/packs/site_styles/_site-style.scss

```diff
@@ -26,20 +26,49 @@ body {
     &.dark{
     background: tint(#0a0527, 10);
     }
-    .mobile-popover-nav {
-      display: none;
+    .kit-search {
+      width: 33vw;
+      max-width: 300px;
+      [class*=_label] {
+        margin-bottom: 0 !important;
+      }
+      .text_input_wrapper {
+        margin-bottom: 0 !important;
+      }
+    }
+    .nav-links-mg-override {
+      margin: 0 !important;
+    }
+    .pb-version-tag {
+      text-align: right;
+      position: relative;
+      bottom: $space_md - 4;
+    }
+    .separator-explicit-height {
+      height: $space_xl + 5;
+      margin-left: $space_xs;
+      margin-right: $space_xs;
+    }
+    .toggle-icon {
+      color: $text_lt_light;
```

Comment:
![image](https://user-images.githubusercontent.com/863031/101692049-38780700-3a35-11eb-9da1-1c299f3ce5b0.png)

Should be lighter. Why not use a body kit do you don't have to set this?

Reply:
Can we also make the icon clickable? Let me know if that would be easy or not

Reply:
Good call, didn't think to wrap icon in body kit.

Reply:
Can make the icon clickable (wrap in a tag with href pointed to dark mode method routes) but it won't trigger the toggle on click.  Might confuse users.

Reply:
Actually I'll see if there's a way to trip the toggle with javascript and create an onClick handler for the icon.

Reply:
Yep that works.